docs/installation.rst

```diff
@@ -238,6 +226,36 @@ this may come up empty. To fix it, run the following::
 
     for i in /usr/local/lib/python2.7/site-packages/*mesos*; do ln -snf $i venv/lib/python2.7/site-packages/ ; done
 
+.. _docker-note:
+.. topic:: Installing Docker
```

Comment:
This treats Docker and Quay and the heading should reflect that. So should the sentence above that links to this note.